src/ResourceManager/IotHub/Commands.IotHub/IotHub/SetAzureRmIotHub.cs

```diff
@@ -122,6 +126,27 @@ public class SetAzureRmIotHub : IotHubBaseCmdlet
         [ValidateNotNullOrEmpty]
         public PSOperationsMonitoringProperties OperationsMonitoringProperties { get; set; }
 
+        [Parameter(
+            ParameterSetName = UpdateRoutingPropertiesParameterSet,
+            Mandatory = false,
+            HelpMessage = "RoutingProperties")]
+        [ValidateNotNullOrEmpty]
+        public PSRoutingProperties RoutingProperties { get; set; }
```

Comment:
I changed the type name for RouteProperties , are you suggesting changing this too ? So we have the following now ...
PSRoutingProperties
PSRouteMetadata
PSFallbackRouteMetadata
PSEndpoints 
Please suggest if you want these to be changed...